Classes/Controller/Backend/NewTenantController.php

```diff
@@ -419,8 +419,10 @@ protected function getLLL($index, $lang, $langArray)
     {
         if (isset($langArray[$lang][$index][0]['target'])) {
             return $langArray[$lang][$index][0]['target'];
-        } else {
+        } else if (isset($langArray['default'][$index][0]['target'])) {
             return $langArray['default'][$index][0]['target'];
+        } else {
+            return "Missing translation for ".substr($index, 9); //cut 'metadata.' string
```

Comment:
Since this method is not only used for metadata, but also for structure labels, the `substr()` function would not work as intended. I'd be fine with just using the whole translation key here:
```suggestion
            return 'Missing translation for ' . $index;
```
What do you think?

Reply:
Ok I didn't knew that. Your suggestion looks fine to me.